extensions/ql-vscode/src/remote-queries/shared/variant-analysis.ts

```diff
@@ -34,6 +34,7 @@ export enum VariantAnalysisStatus {
   InProgress = 'inProgress',
   Succeeded = 'succeeded',
   Failed = 'failed',
+  Canceled = 'canceled',
```

Comment:
Just a thought, but likely something outside of this PR: should we also allow timed out status?

Reply:
It would make sense, but I don't think we currently have a design for the "timed out" status. Do we need to distinguish it from the failed status?

Reply:
I think it's still a failure, but we will be showing a message saying that things timed out. Let's just have that in mind for now and see how things go.